packages/aws-library/src/aws_library/ec2/_client.py

```diff
@@ -61,13 +62,13 @@ async def close(self) -> None:
         await self.exit_stack.aclose()
 
     async def ping(self) -> bool:
-        try:
```

Comment:
MINOR: This is perfectly valid but I think a try-catch here is actually more clear.